recipes/ctds/meta.yaml

```diff
@@ -14,26 +14,33 @@ source:
 build:
   number: 0
   features:
-    - vc9  # [win and py27]
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
     - vc14  # [win and py >= 35]
   script:
-    - python setup.py build_ext --include-dirs=$PREFIX/include --library-dirs=$PREFIX/lib
+    - python setup.py build_ext --include-dirs=%PREFIX%\include --library-dirs=%PREFIX%\lib  # [win]
+    - python setup.py build_ext --include-dirs=$PREFIX/include --library-dirs=$PREFIX/lib    # [not win]
     - python setup.py install --single-version-externally-managed --record record.txt
+  # Due to this requiring stdbool.h and windows C compilers not having C99 Headers
+  # disable for now
+  skip: true  # [win]
```

Comment:
@mariusvniekerk take a look at this nice solution from @gillins 

https://github.com/conda-forge/expat-feedstock/pull/11/commits/505b5ae08b9445b00f078342fb012ef9b8169af3